EXILED/Exiled.API/Features/Npc.cs

```diff
@@ -63,6 +64,101 @@ public override Vector3 Position
             }
         }
 
+        /// <summary>
+        /// Gets or sets the player being followed.
+        /// </summary>
+        /// <remarks>The npc must have <see cref="PlayerFollower"/>.</remarks>
+        public Player? FollowedPlayer
+        {
+            get => !GameObject.TryGetComponent(out PlayerFollower follower) ? null : Player.Get(follower._hubToFollow);
+
+            set
+            {
+                if (!GameObject.TryGetComponent(out PlayerFollower follower))
+                    return;
+
+                follower._hubToFollow = value?.ReferenceHub;
```

Comment:
Are you sure that you can't add this component if new target is not null

Reply:
i can but idk